FirebaseMessaging/Sources/Public/FIRMessaging.h

```diff
@@ -365,20 +365,47 @@ NS_SWIFT_NAME(Messaging)
 @property(nonatomic, assign, getter=isAutoInitEnabled) BOOL autoInitEnabled;
 
 /**
- *  The FCM token is used to identify this device so that FCM can send notifications to it.
- *  It is associated with your APNS token when the APNS token is supplied, so that sending
- *  messages to the FCM token will be delivered over APNS.
+ * The FCM registration token is used to identify this device so that FCM can send notifications to
+ * it. It is associated with your APNS token when the APNS token is supplied, so that sending
+ * messages to the FCM token will be delivered over APNS.
  *
- *  The FCM token is sometimes refreshed automatically. In your FIRMessaging delegate, the
- *  delegate method `messaging:didReceiveRegistrationToken:` will be called once a token is
- *  available, or has been refreshed. Typically it should be called once per app start, but
- *  may be called more often, if token is invalidated or updated.
+ * The FCM registration token is sometimes refreshed automatically. In your FIRMessaging delegate,
+ * the delegate method `messaging:didReceiveRegistrationToken:` will be called once a token is
+ * available, or has been refreshed. Typically it should be called once per app start, but
+ * may be called more often, if token is invalidated or updated.
  *
- *  Once you have an FCM token, you should send it to your application server, so it can use
- *  the FCM token to send notifications to your device.
+ * Once you have an FCM token, you should send it to your application server, so it can use
+ * the FCM token to send notifications to your device.
  */
 @property(nonatomic, readonly, nullable) NSString *FCMToken NS_SWIFT_NAME(fcmToken);
 
+/**
+ * Asynchronously getting the default FCM registration token.
+ *
+ * This creates a Firebase Installations ID, if one does not exist, and sends information
```

Comment:
How does the Installations ID relate to the FCM registration token?

Reply:
Actually I just realize this call doesn't necessarily creates a FID, FID is always existed in the client.  @gsakakihara 
So I don't think we need to state here what FID's impact on user info collecting, but we should mention the registration token's impact on this.
@andirayo @egilmorez

Reply:
Looks like FIS has changed to not auto generated anymore #5656 to keep consistent behavior between android and iOS. The installations ID is only created if some product or developers request it.